packages/components/src/color-palette/index.tsx

```diff
@@ -257,43 +254,48 @@ function UnforwardedColorPalette(
 					isRenderedInSidebar={ __experimentalIsRenderedInSidebar }
 					renderContent={ renderCustomColorPicker }
 					renderToggle={ ( { isOpen, onToggle } ) => (
-						<Flex
-							as={ 'button' }
-							ref={ customColorPaletteCallbackRef }
-							justify="space-between"
-							align="flex-start"
-							className="components-color-palette__custom-color"
-							aria-expanded={ isOpen }
-							aria-haspopup="true"
-							onClick={ onToggle }
-							aria-label={ customColorAccessibleLabel }
-							style={
-								showTransparentBackground( value )
-									? { color: '#000' }
-									: {
-											background: value,
-											color:
-												colordColor.contrast() >
-												colordColor.contrast( '#000' )
-													? '#fff'
-													: '#000',
-									  }
+						<VStack spacing={ 0 }>
+							<button
+								ref={ customColorPaletteCallbackRef }
+								className="components-color-palette__custom-color-button"
+								aria-expanded={ isOpen }
+								aria-haspopup="true"
+								onClick={ onToggle }
+								aria-label={ customColorAccessibleLabel }
+								style={
+									showTransparentBackground( value )
+										? {}
+										: {
+												background: value,
+										  }
+								}
```

Comment:
Probably we could add to this PR the change to always show a checkered background.

The idea is to have the checkered background always drawn and to render on top the chosen color. Therefore, unless there is a 100% opaque selected color, the checkered background would partially show.

Reply:
That was fun. I'm glad you and I had that chat about stacking contexts recently 😆. I've updated the PR to make the checkered background always show through the chosen color (until it's fully opaque, of course). Let me know what you think!